src/formio/components/AddressNL.js

```diff
@@ -36,6 +36,7 @@ export default class AddressNL extends Field {
         defaultValue: {},
         validateOn: 'blur',
         deriveAddress: false,
+        layout: {show: 'doubleColumn'},
```

Comment:
```suggestion
        layout:  'doubleColumn',
```
there is no `show` (or need for it) in the types :)